src/serializer.ts

```diff
@@ -88,6 +93,8 @@ export class DocxSerializerState<S extends Schema = any> {
     this.options = options ?? {};
     this.children = [];
     this.numbering = [];
+
+    this.options.fontSize = 17;
```

Comment:
Should we have a `DEFAULT_FONT_SIZE` const somewhere? I noticed you defaulted to 17 in `cssToDocxStyle.ts ` as well.

Reply:
yes, I'll make one.